pkgs/development/python-modules/jq/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage, fetchPypi, lib, cython, jq }:
+
+buildPythonPackage rec {
+  pname = "jq";
+  version = "0.1.6";
+
+  srcs = fetchPypi {
+    inherit pname version;
+    sha256 = "34bdf9f9e49e522e1790afc03f3584c6b57329215ea0567fb2157867d6d6f602";
+  };
+  patches = [./jq-py-setup.patch];
+
+  nativeBuildInputs = [cython];
+
+  preBuild = ''
+    cython jq.pyx
+  '';
+
+  propagatedBuildInputs = [jq];
```

Comment:
Why propagate? I would expect the compiled code to have a reference to `jq`.

Reply:
You're right, it does.  I'll change this to `buildInputs`; I suspect the only difference will be in the contents of `$out/nix-support/propagated-build-inputs`